Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:alpine3.11
+
+COPY . /app
+WORKDIR /app
+
+RUN yarn && yarn build
+
+FROM nginx:stable-alpine
+LABEL author="Benjamin Guillon <bengrunt@gmail.com>"
```

Comment:
That is some shameless name dropping! When this game is a hit, you'll be riding on its fame!

Reply:
Fixed.